ucm2/sof-soundwire/rt722.conf

```diff
@@ -0,0 +1,97 @@
+# Use case Configuration for sof-soundwire card
+
+If.codecmic {
+	Condition {
+		Type RegexMatch
+		Regex "(rt722(-sdca)?)"
+		String "${var:MicCodec1}"
```

Comment:
The `MicCodec1` variable may be empty when `MultiCodec1` is set. You should define/use another variable to make this decision.